Update to WebSub Terminology (#450)
HL7/fhircast-docs: input/pagecontent/4-1-launch-scenarios.md

This page contains guidance to implementers and is not part of the [normative-track](2_Specification.html). 
<p></p><p></p>

> A FHIRcast Hub uses a unique `hub.topic` session id to identify a single session across the Hub, Subscribers and driving applications (also Subscribers to the topic) which are engaged in the shared session. The `hub.topic` must be known by an application for it to subscribe to the session. Typically, the Hub defines the `hub.topic`.

Reviewer: ```suggestion
A FHIRcast Hub uses a unique `hub.topic` session id to identify a single session across the Hub, subscribing and driving applications (i.e. Subscribers) which are engaged in the shared session. The `hub.topic` must be known by an application for it to subscribe to the session. Typically, the Hub defines the `hub.topic`.

```
